docs/other-topics/legacy.md

```diff
@@ -54,6 +54,24 @@ Collection.init({
 
 And if your model has no primary key at all you can use `Model.removeAttribute('id');`
 
+Instances without primary keys can still be retrieved using `Model.findOne` and `Model.findAll`.  
+While it's currently possible to use their instance methods (`instance.save`, `instance.update`, etc…), doing this will lead to subtle bugs, 
+and is planned for removal in a future update.
+
+:::info
```

Comment:
oh what is this syntax doing?

Reply:
Admonitions! https://docusaurus.io/docs/next/markdown-features/admonitions